### DISCUSSION TITLE
Problem with force setting in setJointMotorControl2/Array

### DISCUSSION THREAD

**Discussion by tianhaowuhz** [bulletphysics/bullet3#3383] [General]
I noticed that in the example of control panda robot, for the force setting, it use"force=5*240.",is there any specific reason why times "240." in here because if I only use "5" it seems not have enough force for the arm.

**Comment by rohit-kumar-j** [comment]
@moistchi, please link the specific file that you are referring to.

Warm Regards,
Rohit

**Comment by tianhaowuhz** [comment]
> @moistchi, please link the specific file that you are referring to.
> 
> Warm Regards,
> Rohit
it's in panda_sim.py
https://github.com/bulletphysics/bullet3/blob/afa4fb54505fd071103b8e2e8793c38fd40f6fb6/examples/pybullet/gym/pybullet_envs/examples/panda_sim.py
